CHANGES

```diff
@@ -1,3 +1,7 @@
+## 1.7.2
```

Comment:
should say `next` ideally since it's not fixed what version this will be in, but since nothing else is changing probably safe